gen3-load-tests/download_performance_test.py

```diff
@@ -0,0 +1,2226 @@
+#!/usr/bin/env python3
+"""
+Compares CDIS Data Client and Gen3 SDK download-multiple-async functionality
+Performs REAL downloads using provided GUIDs and shows detailed performance metrics
+"""
+
+import json
+import logging
+import os
+import subprocess
+import time
+import threading
+import asyncio
+import sys
+import argparse
+import html
+import shutil
+import webbrowser
+import math
+import cProfile
+import pstats
+import io
+import zipfile
+from datetime import datetime
+from typing import Dict, List, Any, Optional, Tuple
+from dataclasses import dataclass, field
+from statistics import mean, stdev
+
+from contextlib import contextmanager
+
+try:
+    import psutil
+
+    PSUTIL_AVAILABLE = True
+except ImportError:
+    PSUTIL_AVAILABLE = False
+    print("Warning: psutil not available. Install with: pip install psutil")
+
+try:
+    import sys
+    import subprocess
+    import os
+
+    gen3sdk_path = os.path.expanduser(
+        "~/path/to/gen3sdk-python"
+    )  # TODO: Update this path
```

Comment:
we can't have anything in here that requires updating the code and you absolutely do not want to be doing path modifications in here. Every dev is going to have diff python versions, manage venv differently, you can't hard-code this kind of path manipulation. 

Instead, we can allow installing gen3 from a branch in this venv and remove all this code from line 44-60.

Instead update the docs to specify that you can test a branch of gen3 by updating the pyproject.toml to something like:

```
gen3 = {git = "https://github.com/Dhiren-Mhatre/gen3sdk-python", branch = "feat/multiple-download-performance-testing"}
```

and then poetry lock / poetry install

Reply:
try it this way and adjust the code to work like that. We want to avoid any paths or manipulation in this code. You should just inform the tester to update their pyproject.toml to install the gen3 SDK version they want to test